src/components/MarkdownProvider/components/PropSheets.tsx

```diff
@@ -6,36 +6,49 @@
  */
 
 import React from 'react';
-import { css } from 'styled-components';
+import styled from 'styled-components';
 import { ComponentDoc } from 'react-docgen-typescript';
-import { getColor } from '@zendeskgarden/react-theming';
-import { MD, Paragraph } from '@zendeskgarden/react-typography';
-import { Table, Head, Body, HeaderRow, HeaderCell, Row, Cell } from '@zendeskgarden/react-tables';
+import { Paragraph, Code } from '@zendeskgarden/react-typography';
+import {
+  Table,
+  Head,
+  Body,
+  HeaderRow,
+  HeaderCell,
+  Row as GRow,
+  Cell
+} from '@zendeskgarden/react-tables';
+import { StyledH4 as Title } from './Typography';
 
-import { IPackage } from './PackageDescription';
-import { StyledH3 } from './Typography';
+const API = styled.div`
+  margin: ${p => p.theme.space.base * 6}px 0;
```

Comment:
Rather than use `base * x` math, let's use the defined theming sizes: `sm`, `md`, `lg`, etc.